frontend/src/pages/ProgressionChecker/ProgressionChecker.jsx

```diff
@@ -41,16 +46,57 @@ const degreeData = {
   ],
 };
 
+const ProgressionCheckerCourses = ({ structure, isLoading }) => {
+  const DEFAULTMODE = "grid-concise";
+  const [view, setView] = useState(DEFAULTMODE);
+
+  return (
+    <div>
+      {(view === "grid" || view === "grid-concise") ? (
```

Comment:
Think its better to use enums here? Or compare it to a constant variable instead!

Reply:
we stan